### ISSUE TITLE
server errorstat_ERR spike in version 4.4.0

### ISSUE THREAD

**Issue by rainecp** [redis/jedis#3648]
We recently upgraded our jedis library from  v4.1.1 to v4.4.6 and noticed a spike in errorstat_ERR on our redis servers. This seems to be internal as we don't get the error in the client response. I was able to track down the version where this anomaly was introduced which at 4.4.0.

### Expected behavior

When invoking a redis get or set command, there is no increase in errorstat_ERR if the client call was successful.

### Actual behavior

When invoking redis commands, we intermittently see spike in errorstate_ERR that is not propagated to the client since we are still getting a successful response. Errors within the server could probably happen from time to time but it is strange that I could consistently see an increase in errorstat when starting our service and calling a jedis command for the first time, it goes up by 2. But this does not happen in version 4.1.1 up to 4.3.2. I tried using both JedisPool and JedisPooled, called 200x both have an errorstat of 4 in the new version and 0 in 4.1.1.

### Steps to reproduce:

1. Using jedis 4.4.0 or higher, instantiate a JedisPool("127.0.0.1", 6379) (or JedisPooled)
2. Invoke a SET command for the first time
3. Check the metrics and notice errorstat increased by 2
4. Using an older version, repeat steps 1-2, errorstat is empty

**Comment by sazzad16** [comment]
It must be due to internal CLIENT SETINFO commands.
You can set ClientSetInfoConfig#isDisabled() to true in JedisClientConfig. `ClientSetInfoConfig.DISABLED` can be used for this.

**Comment by rainecp** [comment]
Thanks for the quick response. I confirmed this is the issue, after setting `ClientSetInfoConfig.DISABLED`, I don't see the up tick in `errorstat_ERR` anymore. When directly executing CLIENT SETINFO command using `redis-cli`, I also get an error which is expected since based on [this](https://redis.io/commands/client-setinfo/), it was introduced recently in 7.2 and would fail on older version.  I also tested the same command against redis:7.2-alpine docker and I don't get the error.
